cpp/unreal_plugins/WheeledVehicle/WheeledVehicle.uplugin

```diff
@@ -27,7 +27,7 @@
             "Enabled": true
         },
         {
-            "Name": "PhysXVehicles",
+            "Name": "ChaosVehiclesPlugin",
```

Comment:
Why do these plugins not match the Build.cs file?

Reply:
The plugin name here is different (`ChaosVehiclesPlugin`) from the module names (`ChaosVehicles`, `ChaosVehiclesCore`). Since, a plugin can have multiple modules, module names always don't match the plugin name.

Reply:
Makes sense.